app/about/page.js

```
@@ -0,0 +1,185 @@
+                    </div>
+                </div>
+
+                <div className="bg-blue-50 p-10 rounded-xl text-center mb-12">
+                    <h2 className="text-2xl font-bold text-blue-900 mb-4">Join Us</h2>
+                    <p className="text-lg text-gray-700 mb-6">
+                        Whether you're a BCA student looking to grow your skills or someone passionate
+                        about tech and community building, the BCA Association – MMC welcomes you.
+                        Together, let's learn, lead, and grow.
+                    </p>
+// At the top of app/about/page.js
+import Link from 'next/link';
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)